sre_maturity.md

```diff
@@ -0,0 +1,114 @@
+# SRE Maturity Model
+
+## Phase 1: The ‘Crawl’ Phase
+
+### Crawl Phase Overview
+
+You’re starting from zero, with a net new team.
+You may only have a small SRE team, or an SRE function within your development team.
```

Comment:
I'm not sure that the size of the SRE team necessarily correlates with some of the other measures. In the RHEL org, even small (non-managed-service) teams which are responsible for running their own service (think CI services for internal customers etc) are expected to provide a reliable service, managed according to the later stages of the model.

Reply:
Agreed.
Also, there are other personas, like existing ops or devops teams, that could be at the early phases of maturity in some areas.
I'm going to rearrange the PR so there are phases of maturity in different areas rather than talking about overall maturity